pkg/types/020/types_test.go

```diff
@@ -53,14 +53,14 @@ func testResult(resultCNIVersion, jsonCNIVersion string) (*types020.Result, stri
 			IP:      *ipv4,
 			Gateway: net.ParseIP("1.2.3.1"),
 			Routes: []types.Route{
-				{Dst: *routev4, GW: routegwv4},
+				{Dst: *routev4, GW: routegwv4, MTU: 1024},
```

Comment:
We should probably not add any new fields to v0.2.0 and v0.4.0 -- what do you think, @dcbw ?

Reply:
@Joffref yes please.
(Sorry, I somehow missed your question!)

Reply:
Change made!